lib/config.js

```diff
@@ -5,4 +5,27 @@
 var config = require('bedrock').config;
 var path = require('path');
 
-config['messages-email'] = {};
+config['messages-email'] = config['messages-email'] ||  {};
+config['messages-email'].emailEventType =
+  config['messages-email'].emailEventType || 'bedrock.messageEmailEventType';
+
+config['messages-email'].defaultUserSettings =
+  config['messages-email'].defaultUserSettings ||
+  {enable: true, interval: 'immediate'};
+
+config.mail.events.push({
+  type: 'bedrock.messageEmailEventType',
+  // email for new message
+  template: 'generic.notification.email'
+});
+
+var ids = [
+  'generic.notification.email'
+];
+ids.forEach(function(id) {
+  config.mail.templates.config[id] = {
+    // adjust to point to the module email templates
+    filename: path.join(__dirname, '..', 'email-templates', id + '.tpl')
```

Comment:
We could remove the `bedrock-messages-email.` namespacing prefix from the filename here to keep the filenames simple.